main/Kconfig.projbuild

```diff
@@ -31,32 +31,61 @@ config BELL_INPUT_GPIO
         range 0 39
         default 0
         help
-                GPIO number (IOxx) to read the bell state from.
-                Some GPIOs are used for other purposes (flash connections, etc.) and cannot be used.
-                GPIO 0 is also the boot pin, so the BOOT button can be used for testing.
+            GPIO number (IOxx) to read the bell state from.
+            Some GPIOs are used for other purposes (flash connections, etc.) and cannot be used.
+            GPIO 0 is also the boot pin, so the BOOT button can be used for testing.
 
 config RING_DURATION
     int "Ring duration in milliseconds"
         range 1000 30000
         default 7000
         help
-                This is the maximum duration, the called phone rings.
-		In reality, this phone rings for a shorter duration, because there is a delay, between
-		the SIP ring request, the ACK from the SIP server and the actual ringing of the phone.
+            This is the maximum duration, the called phone rings.
+            In reality, this phone rings for a shorter duration, because there is a delay, between
+            the SIP ring request, the ACK from the SIP server and the actual ringing of the phone.
+        
+config ACTUATOR_OUTPUT_GPIO
```

Comment:
There should be a way to completely disable the actuator output.
Please add a Kconfig variable to enable/disable this feature and only display these added items as a sub-menu.